### ISSUE TITLE
Codegen failing

### ISSUE THREAD

**Issue by MarkMarcel** [vishna/voyager#96]
Kindly have a look at the below image. Can't seem to get codegen to work. I will provide further information on request if needed

![Screenshot (10)](https://user-images.githubusercontent.com/45716888/113917184-69442f00-97d0-11eb-8afc-a4035ce6de10.png)

Continuation

![Screenshot (11)](https://user-images.githubusercontent.com/45716888/113917826-39495b80-97d1-11eb-8abe-08e82403a8f0.png)

**Comment by vishna** [comment]
Ah Windows 😅

So... next to pubspec.yaml, do you have `voyager-codegen.yaml` file? If so, what are the contents of `voyager-codegen.yaml`?

**Comment by MarkMarcel** [comment]
Yes, I have voyager-codegen.yaml at the root of the project. However, I had to manually create it as the tool couldn't create it. Every time I tried, it got stuck.

Here is my folder structure and contents of voyager-codegen.yaml

![Screenshot (12)](https://user-images.githubusercontent.com/45716888/114148975-e7582100-9909-11eb-83c8-fdf8730fa8e5.png)

**Comment by vishna** [comment]
Could you maybe git check out the voyager/example folder and see if codegen works when run from inside that folder? My initial bet is on the path and handling of spaces/slashes under Windows. Maybe codegen doesn't like spaces in the "Interactive Edge Dev" 🤷

**Comment by MarkMarcel** [comment]
Sure I will do that. 

Is possible that you can handle such an edge case in the package as well?
